src/main/java/com/marklogic/appdeployer/command/schemas/LoadSchemasDataCommand.java

```diff
@@ -0,0 +1,51 @@
+package com.marklogic.appdeployer.command.schemas;
+
+import java.io.File;
+
+import com.marklogic.appdeployer.AppConfig;
+import com.marklogic.appdeployer.command.AbstractCommand;
+import com.marklogic.appdeployer.command.CommandContext;
+import com.marklogic.appdeployer.command.SortOrderConstants;
+import com.marklogic.client.DatabaseClient;
+import com.marklogic.client.dataloader.SchemasDataLoader;
+import com.marklogic.client.dataloader.impl.DefaultSchemasDataFinder;
+import com.marklogic.client.dataloader.impl.DefaultSchemasDataLoader;
+
+public class LoadSchemasDataCommand extends AbstractCommand  {
+
+	private SchemasDataLoader schemasDataLoader;
+	
+	public LoadSchemasDataCommand() {
+        setExecuteSortOrder(SortOrderConstants.DEPLOY_SQL_VIEWS);
```

Comment:
You're right -- there could be modules that depend on shemas.  I'll figure out where it goes, or make a new key for schemas